zerver/tests/tests.py

```diff
@@ -166,6 +166,42 @@ def test_do_deactivate_realm(self):
         user = get_user_profile_by_email('hamlet@zulip.com')
         self.assertTrue(user.realm.deactivated)
 
+    def test_do_set_realm_default_language(self):
+        # type: () -> None
+        new_lang = "de"
+        realm = get_realm('zulip.com')
+        self.assertNotEqual(realm.default_language, new_lang)
+        email = 'cordelia@zulip.com'
+        self.login(email)
+        user_profile = get_user_profile_by_email(email)
+        do_change_is_admin(user_profile, True)
```

Comment:
if you use `iago@zulip.com`, that user is an admin by default, so you can cut a bit of code here.